src/page-objects/storefront/AccountAddresses.ts

```diff
@@ -9,7 +9,7 @@ export class AccountAddresses implements PageObject {
     public readonly useDefaultShippingAddressButton: Locator;
 
     constructor(public readonly page: Page) {
-        this.addNewAddressButton = page.getByRole('link', { name: 'Add new address' });
+        this.addNewAddressButton = page.getByRole('link', { name: /Add (new )?address/ });
```

Comment:
What happend here?

Reply:
different snippet in 6.5 `Add new address`, so we match both